BREAKING.md

```diff
@@ -1,3 +1,18 @@
+## 0.46 Breaking changes
+- [FrsClient has been renamed to AzureClient](#FrsClient-has-been-renamed-to-AzureClient)
```

Comment:
nit: the # link needs to be all lowercase to work i think

Reply:
The `Property-removed-from-IFluidDataStoreRuntime` in the 0.45 changes has some uppercase but looks like its still working